src/waveform/renderers/waveformmarkset.cpp

```diff
@@ -22,21 +22,14 @@ void WaveformMarkSet::setup(const QString& group, const QDomNode& node,
     bool hasDefaultMark = false;
 
     QDomNode child = node.firstChild();
-    DefaultMarkerStyle defaultModel;
+    QDomNode defaultChild;
     int priority = 0;
     while (!child.isNull()) {
         if (child.nodeName() == "DefaultMark") {
-            defaultModel = DefaultMarkerStyle{
-                    context.selectString(node, "Control"),
-                    context.selectString(node, "VisibilityControl"),
-                    context.selectString(node, "TextColor"),
-                    context.selectString(node, "Align"),
-                    context.selectString(node, "Text"),
-                    context.selectString(node, "Pixmap"),
-                    context.selectString(node, "Icon"),
-                    context.selectString(node, "Color"),
-            };
+            m_pDefaultMark = WaveformMarkPointer(new WaveformMark(
+                    group, child, context, --priority, signalColors));
```

Comment:
NIT (no naked new, allocate two obj and ptr at once):
```suggestion
            m_pDefaultMark = WaveformMarkPointer::create(
                    group, child, context, --priority, signalColors);
```
same below